lib/svg-sprite/mode/base.js

```diff
@@ -182,8 +182,8 @@ SVGSpriteBase.prototype._buildCSSResources = function(files, cb) {
         };
     };
 
-    for (const extension in this.config.render) {
-        tasks.push(createResourceTask(this.config.render[extension], this.data, this._spriter, extension));
+    for (const [extension, value] of Object.entries(this.config.render || {})) {
```

Comment:
This is the fallback I needed to add. Maybe we could make this branch more explicit with an if?